app/javascript/mastodon/features/direct_timeline/index.js

```diff
@@ -10,7 +10,7 @@ import { connectDirectStream } from '../../actions/streaming';
 import ConversationsListContainer from './containers/conversations_list_container';
 
 const messages = defineMessages({
-  title: { id: 'column.direct', defaultMessage: 'Direct messages' },
+  title: { id: 'column.conversations', defaultMessage: 'Conversations' },
```

Comment:
What about “personal message”? Still convey a similar idea but more reminiscent of BBS and similar communities smaller than Twitter.

Reply:
I give up, Conversations is fine.